calico/felix/frules.py

```diff
@@ -240,15 +240,16 @@ def install_global_rules(config, v4_filter_updater, v6_filter_updater,
         iptables_generator.raw_rpfilter_failed_chain(ip_version=6)
     )
 
-    v6_raw_updater.rewrite_chains({CHAIN_PREROUTING: v6_raw_prerouting_chain},
-                                  {CHAIN_PREROUTING: v6_raw_prerouting_deps},
-                                  async=False)
-
-    v6_raw_updater.ensure_rule_inserted(
-        "PREROUTING --in-interface %s --match rpfilter --invert "
-        "--jump %s" %
-        (iface_match, CHAIN_PREROUTING),
-        async=False)
+    if v6_raw_updater:
```

Comment:
The lines just above this should be moved inside the conditional.  No point creating the `v6_raw...` objects if they're not going to be used.